cmd/oci-create-runtime-bundle/main.go

```diff
@@ -81,10 +87,20 @@ func newBundleCmd(stdout, stderr *log.Logger) *cobra.Command {
 It is strongly recommended to keep the default value.`,
 	)
 
+	cmd.Flags().BoolVar(
+		&v.version, "version", false,
+		`Print version information and exit`,
+	)
 	return cmd
 }
 
 func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
+	if v.version {
+		v.stderr.Printf("commit: %s", gitCommit)
+		v.stderr.Printf("spec: %s", specs.Version)
```

Comment:
@wking 
> I expect image-tools to support a range of spec versions, not just a single one. For example, once image-spec cuts 2.0, image-tools should still be able to process 1.x images for some reasonable deprecation period. image-spec is setting us up for that possibility by using the v1 package for the Go types that are likely to change. And SemVer means that if we can unpack a 1.3 image, we can also unpack 1.2, 1.1, and 1.0 images. So I think we want this to be printing:

I think this is hard. It's not only the `v1` package, the spec define the image layout, how we validate the image, how we unpack the layer... etc, this could be differ from version to version. the tool is a implementation of the spec at some point, the 
````
specs: 0.4.0
````
means the tool is developed based on the v0.4.0 spec. 
After 1.0, I agree to show the supported versions but the tool need to do some backward compatibility thing.

>And if the caller explicitly asked for the version information with --version, I think we should be printing it to stdout instead of stderr.

This is a mistake, will fixed.